main.go

```diff
@@ -72,7 +72,13 @@ func main() {
 	jsonrpc.RegRoutesGroups(appOpRoutes)
 	jsonrpc.PrintRoutes(appOpRoutes)
 
-	if err := r.Run(cfg.URL); err != nil {
-		logrus.Fatal("Unable to start server. Cause: ", err.Error())
+	if cfg.UseTLS {
+		if err := r.RunTLS(cfg.URL, "/var/serving-cert/tls.crt", "/var/serving-cert/tls.key"); err != nil {
```

Comment:
I wonder if it makes sense to configure certs dir instead of useTLS flag with hard-coded path, then chePlugin would be able to configure path like `/che-machine-exec --serving-tls-certs=/var/run/secrets/openshift.io/cluster-tls-certs`, or even `che-machine-exec --serving-tls-certs=$(CLUSTER_TLS_CERTS)` and controller provision value for that env vars. It would be more consistent but I cannot say simpler =)